packages/core/src/App/Containers/RealAccountSignup/address-details-form.js

```diff
@@ -19,7 +19,16 @@ const address_details_config = ({ account_settings }) => {
         address_line_2: {
             supported_in: ['svg', 'iom', 'malta', 'maltainvest'],
             default_value: account_settings.address_line_2 ?? '',
-            rules: [['length', localize('Address line 2 is not in a proper format'), { min: 0, max: 30 }]],
+            rules: [
+                ['length', localize('Address line 2 is not in a proper format'), { min: 0, max: 30 }],
```

Comment:
shouldn't the message be specific to the length here? would be more clear for the user